docs/sources/architecture/_index.md

```diff
@@ -88,7 +88,7 @@ value is larger than the hash of the stream. When the replication factor is
 larger than 1, the next subsequent tokens (clockwise in the ring) that belong to
 different ingesters will also be included in the result.
 
-The effect of this hash set up is that each token that an ingester owns is
+The effect of this hash setup is that each token that an ingester owns is
```

Comment:
This version of "setup" is a noun. Please revert this change.

Reply:
Obviously happy to revert this, but shouldn't it be a noun?
Or should it be "The effect of this hash being set up is that..."?

Reply:
gotcha, apologies! Cheers for the feedback, I've reverted that change

Reply:
If you are ready for another review, I suggest clicking "Re-request review." People don't always notice comments, but the request for review sends a separate (and more visible) notification.

Reply:
Thanks, didn't know that!